conf/web.conf

```diff
@@ -168,6 +168,8 @@ username =
 password =
 guacd_host = localhost
 guacd_port = 4822
+# Server that exposes the VNC ports (e.g., your KVM host)
```

Comment:
no need to edit those files, only .default, i can't remove that by myself as i don't have rights